src/app/app.component.html

```diff
@@ -1 +1,3 @@
-<router-outlet></router-outlet>
+<app-navbar>
```

Comment:
Why you're using `<router-outlet>` inside `app-navbar`? `app-navbar` should be separate header component and `router-outlet` should display main page content independently

Reply:
Cuz `navbar` consist of `mat-toolbar` on top, `mat-toolbar` on left side and `mat-sidenav-container` for all the content. That's how folks from **material** imagining it should be. Passing `router` because there is `<ng-content></ng-content>` in place.